docs/store/data-items.md

```diff
@@ -10,20 +10,26 @@ of data and metadata.
 Example function:
 
 ```python
+%%writefile data-prep.py
+import mlrun
+
 def prep_data(context, source_url: mlrun.DataItem, label_column='label'):
     # Convert the DataItem to a Pandas DataFrame
     df = source_url.as_df()
     df = df.drop(label_column, axis=1).dropna()
     context.log_dataset('cleaned_data', df=df, index=False, format='csv')
 ```
 
-Running the function:
+Creating a project, setting the function into it, defining the URL with the data and running the function:
 
 ```python
+source_url = 'https://s3.wasabisys.com/iguazio/data/iris/iris_dataset.csv'
```

Comment:
Why not get sample path?

Reply:
I just wanted the code to work, so I needed to supply a link to some data, this is not a big file, so I think it fits the need.

Reply:
That code will work with mlrun get sample path and it allows us to change the location of data later. Please don't include the full path.

Reply:
Fixed